apache/trafficcontrol
# Fixed job route permission

<hr/>

## Which Traffic Control components are affected by this PR?

- Traffic Ops

## What is the best way to verify this PR?


1. In you dev/traffic_ops/cdn.conf add if not configured or change "role_based_permissions": true to true at the base level.
2. Create or modify a "Portal" user/role and remove the permission DELIVERY-SERVICE:UPDATE.
3. Using your new/modified user, in TP, create a Invalidation Request and verify its creation.
4. Send GET, UPDATE, and DELETE created Invalidation Request. Ensure all works as expected.
5. Verify this all works on v4 and v5 apis
6. When done, insert permission DELIVERY-SERVICE:UPDATE back to the portal role.
7. Create or use an existing user with Read-Only role, and log into Traffic Portal
8. Click to open Delivery Services section, and ensure it works.

## If this is a bugfix, which Traffic Control versions contained the bug?

8.0.0

## PR submission checklist
